tests/acceptance/test_snapshot.py

```diff
@@ -159,11 +164,20 @@ def test_snapshot_using_mender_artifact(self, bitbake_variables, connection):
             # mender-artifact prefixes each ssh argument with "-S"
             common_args = common_args.replace(" ", " -S ")
 
-            subprocess.check_call(
-                "mender-artifact write rootfs-image -S %s -n test -t test -o test_snapshot_using_mender_artifact.mender -f ssh://%s@%s:%s"
-                % (common_args, connection.user, connection.host, connection.port),
-                shell=True,
-            )
+            try:
+                subprocess.check_call(
+                    "%s mender-artifact write rootfs-image -S %s -n test -t test -o test_snapshot_using_mender_artifact.mender -f ssh://%s@%s:%s"
+                    % (
+                        terminal,
+                        common_args,
+                        connection.user,
+                        connection.host,
+                        connection.port,
+                    ),
+                    shell=True,
+                )
+            finally:
+                subprocess.call("cat screenlog.0 ; rm -f screenlog.0", shell=True)
```

Comment:
Can we be sure that `screenlog.0` remains constant?

Reply:
You're right, there's an argument, I'll change to that!